examples/handling-buckets/create-bucket.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+include __DIR__.'/../../vendor/autoload.php';
```

Comment:
We should adjust the shared lines ~ 3 through 12 to be in an `header.php` that is included into each example file.